pkgs/tools/networking/openconnect/default.nix

```diff
@@ -1,6 +1,7 @@
-{ lib, stdenv, fetchurl, pkg-config, openssl ? null, gnutls ? null, gmp, libxml2, stoken, zlib, fetchgit, darwin } :
+{ lib, stdenv, fetchurl, pkg-config, openssl ? null, gnutls ? null, gmp, libxml2, p11-kit, libp11, stoken, zlib, fetchgit, darwin } :
 
 assert (openssl != null) == (gnutls == null);
+assert (openssl != null) -> (libp11 != null);
```

Comment:
```suggestion
```

we don't need this. libp11 cannot be null.

Reply:
I've now made it null when build with openssl

Reply:
We should use boolean inputs instead of overwriting packages to null